kubeless-rbac-novols.jsonnet

```diff
@@ -0,0 +1,9 @@
+# Remove volumeClaimTemplates from kafkaSts to enable testing kubeless
```

Comment:
They maybe, but tbh I don't think it's worth putting time on that, given the ephemeral nature if _this_ use case (i.e. not targeting kafka stateful "endurance" testing) .